server/sonar-server/src/test/java/org/sonar/server/user/ws/SearchActionTest.java

```diff
@@ -58,54 +58,54 @@ public void setUp() {
 
   @Test
   public void search_empty() throws Exception {
-    tester.newGetRequest("api/users", "search2").execute().assertJson(getClass(), "empty.json");
+    tester.newGetRequest("api/users", "search").execute().assertJson(getClass(), "empty.json");
```

Comment:
I have a problem with the SearchAction classes. Don't you think the domain should specified ? Like UserSearchAction ? No need to take an action now but I'd be interested by your POV